src/util/parseDocument.ts

```diff
@@ -95,6 +141,7 @@ const getStyleAttribute = (element: JSXElement): IStyleAttribute | null => {
     ) as ObjectProperty[]
   ).map(property => {
     let value;
+    const key = (property.key as Identifier).name as string;
```

Comment:
Would you be so kind to switch value and key declaration and add a space between them? Otherwise looks good :)

Reply:
I've amended the commit and pushed